eventmesh-common/src/main/java/org/apache/eventmesh/common/config/convert/converter/ObjectConverter.java

```diff
@@ -0,0 +1,200 @@
+
+import org.apache.eventmesh.common.config.Config;
+import org.apache.eventmesh.common.config.ConfigFiled;
+import org.apache.eventmesh.common.config.ConfigInfo;
+import org.apache.eventmesh.common.config.convert.ConvertInfo;
+import org.apache.eventmesh.common.config.convert.ConvertValue;
+import org.apache.eventmesh.common.config.convert.ConverterMap;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.Objects;
+
+import org.assertj.core.util.Strings;
+
+/**
+ * Config field conversion class for Configuration class
+ */
+public class ObjectConverter implements ConvertValue<Object> {
+
+    private String prefix;
+
+    private ConvertInfo convertInfo;
+
+    private Object object;
+
+    private char hump;
+
+    private Class<?> clazz;
+
+    private void init(ConfigInfo configInfo) {
+        String prefix = configInfo.getPrefix();
+        if (Objects.nonNull(prefix)) {
+            this.prefix = prefix.endsWith(".") ? prefix : prefix + ".";
+        }
+        this.hump = Objects.equals(configInfo.getHump(), ConfigInfo.HUMP_ROD) ? '_' : '.';
+        this.clazz = convertInfo.getClazz();
+        this.convertInfo.setHump(this.hump);
+    }
+
+    @Override
+    public Object convert(ConvertInfo convertInfo) {
+        try {
+            this.convertInfo = convertInfo;
+            this.object = convertInfo.getClazz().newInstance();
+            this.init(convertInfo.getConfigInfo());
+            this.setValue();
+
+            Class<?> superclass = convertInfo.getClazz();
+            for (; ; ) {
+                superclass = superclass.getSuperclass();
+                if (Objects.equals(superclass, Object.class) || Objects.isNull(superclass)) {
+                    break;
+                }
+
+                this.clazz = superclass;
+                this.prefix = null;
+                Config[] configArray = clazz.getAnnotationsByType(Config.class);
+                if (configArray.length != 0 && !Strings.isNullOrEmpty(configArray[0].prefix())) {
+                    String prefix = configArray[0].prefix();
+                    this.prefix = prefix.endsWith(".") ? prefix : prefix + ".";
+                    this.hump = Objects.equals(configArray[0].hump(), ConfigInfo.HUMP_ROD) ? '_' : '.';
+                    this.convertInfo.setHump(this.hump);
+                }
+
+                this.setValue();
+            }
+
+            return object;
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private void setValue() throws Exception {
+        boolean needReload = Boolean.FALSE;
+
+        for (Field field : this.clazz.getDeclaredFields()) {
+            if (Modifier.isStatic(field.getModifiers())) {
+                continue;
+            }
+            field.setAccessible(true);
```

Comment:
Here should save the last accessible status and recovery it after custom operation.

Reply:
Done! six six six!